aten/src/ATen/native/mps/kernels/Pooling.h

```diff
@@ -9,25 +9,26 @@ template <unsigned N = 5>
 struct PoolingParams {
```

Comment:
Why not make it a template argument, because we probably want to have something like that for really large tensors

Reply:
Done